technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,29 @@
+# Performance Testing of TEAMMATES
+
+This report gives a detailed explanation of the profiling operations performed on TEAMMATES. It gives an outline of the
+problem and describes our proposed solution.
+
+* [Introduction](#Introduction)
+* [Problem](#Problem)
+* [A brief overview of the Proposed Solution](#Overview-of-Solution)
+* [Tools considered for Performance Testing](#Tools-considered-for-Performance-Testing)
+* [Reasons for using JMeter](#Reasons-for-using-JMeter)
+* [Design of the Workflow](#Design-of-the-workflow)
```

Comment:
Actually, can we merge this section and `overview of the Proposed Solution`? Or do you think it's better to start off explaining what the plan was, and then show what it ended up being?

Reply:
Yeah, I feel it's better to have two separate sections.